src/components/TimelineNav/TimelineNav.module.css

```diff
@@ -13,14 +13,15 @@
   width: 100%;
   overflow: hidden;
   transition: transform var(--eds-anim-fade-long) var(--eds-anim-ease);
-  background: var(--eds-theme-color-background-neutral-default);
-
+  
   @media all and (min-width: $eds-bp-lg) {
-    transition: unset;
+    background: var(--eds-theme-color-background-neutral-default);
+    border-bottom-left-radius: var(--eds-border-radius-xl);
+    border-top-left-radius: var(--eds-border-radius-xl);
     border-width: var(--eds-theme-border-width);
-    border-style: solid;
     border-color: var(--eds-theme-color-border-neutral-subtle);
-    border-radius: var(--eds-border-radius-md);
+    border-style: none;
```

Comment:
Why have a border width and color if the style is none? Could we just do `border: none;`?

Reply:
Good catch.